doc/sphinx/changes.rst

```diff
@@ -8,6 +8,991 @@ Recent changes
 
    .. include:: ../unreleased.rst
 
+Version 8.20
+------------
+
+Summary of changes
+~~~~~~~~~~~~~~~~~~
+
+Coq version 8.20 adds a new rewrite rule mechanism along with a few
+new features, a host of improvements to the virtual machine, the
+notation system, Ltac2 and the standard library.
+
+We highlight some of the most impactful changes here:
+
+  - :ref:`rewrite_rules`
+
+  - :ref:`primitive strings <pstring_library>`
+
+  - A lot of work went into reducing the size of the bytecode segment,
+    which in turn means that .vo files might now be considerably
+    smaller.
+
+  - A new version of the Docker keeper tool to build and maintain
+    Docker images of Coq.
+
+Notable breaking changes:
+
+  - Syntactic global references passed through the `using` clauses of
+    :tacn:`auto`-like tactics are now handled as plain references
+    rather than interpreted terms. In particular, their typeclass
+    arguments will not be inferred. In general, the previous behaviour
+    can be emulated by replacing `auto using foo` with `pose proof
+    foo; auto`.
+
+  - Argument order for the Ltac2 combinators `List.fold_left2` and
+    `List.fold_right2` changed to be the same as in OCaml.
+
+  - :cmd:`Import`\ing a module containing a mutable Ltac2 definition
+    does not undo its mutations. Replace `Ltac2 mutable foo :=
+    some_expr.` with `Ltac2 mutable foo := some_expr. Ltac2 Set foo :=
+    some_expr.` to recover the previous behaviour.
+
+  - Some :ref:`renaming <820_renaming_stdlib>` in the standard
+    library. Deprecations are provided for a smooth transition.
+
+See the `Changes in 8.20.0`_ section below for the detailed list of changes,
+including potentially breaking changes marked with **Changed**.
+Coq's `reference manual for 8.20 <https://coq.github.io/doc/v8.20/refman>`_,
+`documentation of the 8.20 standard library <https://coq.github.io/doc/v8.20/stdlib>`_
+and `developer documentation of the 8.20 ML API <https://coq.github.io/doc/v8.20/api>`_
+are also available.
+
+Théo Zimmermann with help from Ali Caglayan and Jason Gross maintained
+`coqbot <https://github.com/coq/bot>`_ used to run Coq's CI and other
+pull request management tasks.
+
+Jason Gross maintained the `bug minimizer <https://github.com/JasonGross/coq-tools>`_
+and its `automatic use through coqbot <https://github.com/coq/coq/wiki/Coqbot-minimize-feature>`_.
+
+Jaime Arias and Erik Martin-Dorel maintained the
+`Coq Docker images <https://hub.docker.com/r/coqorg/coq>`_
+and the Docker Keeper tool used to build and keep those images up to date.
```

Comment:
@Zimmi48 @erikmd could you please check that sentence

Reply:
Thanks @proux01! I'm the sole author/maintainer of docker-keeper, and we may want to highlight that the tool is generic (it is a compiler from a YAML DSL to GitLab-CI child pipelines, also serialized in YAML).

I propose the following sentence, let me know if it looks good to you:
```suggestion
Jaime Arias and Erik Martin-Dorel maintained the
`Coq Docker images <https://hub.docker.com/r/coqorg/coq>`_.
Erik Martin-Dorel maintained the
`docker-keeper <https://gitlab.com/erikmd/docker-keeper>`_ compiler,
used to build and keep those images up to date, which is generic and applicable to projects beyond Coq.
```

Reply:
What about " (note that the tool is not Coq specific)"?
If you prefer compiler to tool, should we also update the other comment?

Reply:
> What about " (note that the tool is not Coq specific)"?

Yes if you want, if you think it's better.

> If you prefer compiler to tool, should we also update the other comment?

Yes, good point :) it'd be more consistent then.